blackbelt/handle_github.py

```diff
@@ -305,11 +306,15 @@ def deploy(pr_url):
     try:
         ticket_id = get_pr_ticket_id(merge_info['description'])
         move_to_deployed(card_id=ticket_id, comment=comment)
-        create_release(ref=merge_info['sha'], payload='', description="Deployed to production")
     except ValueError:
         if click.prompt("Moving card failed. Open PR in browser?", default=True):
             webbrowser.open(merge_info['html_url'])
 
+    try:
+        create_release(ref=merge_info['branch'], payload='', description="Deployed to production")
+    except ValueError:
```

Comment:
not sure whether `ValueError` is raised if `create_release` fails or there are other exceptions - but as you're not doing anything with the error and it's the last command in function and the default for `bb` at the moment is to fail terribly on exceptions, I'd just remove this second try/except completely

however, YMMV - you can introduce better error handling to `bb` if you want